### DISCUSSION TITLE
Cloud

### DISCUSSION THREAD

**Discussion by BlueMarvinC** [mhonert/velvet-chess#9] [Q&A] [answered]
Thanks

**Comment by mhonert** [comment, marked as answer]
Hi, thanks for your interest in the Velvet Chess Engine!

Do you mean something like the ChessBase Engine Cloud?
As far as I understand, it would require some spare compute resources and a ChessBase installation to provide an engine there.
Unfortunately, I currently have neither available.

Best regards
Martin
